src/components/svgs/lines/styles/chengdumetro-outside-fare-gates.tsx

```diff
@@ -0,0 +1,53 @@
+import { RmgFields, RmgFieldsField } from '@railmapgen/rmg-components';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { AttrsProps } from '../../../../constants/constants';
+import { LinePathAttributes, LinePathType, LineStyle, LineStyleComponentProps } from '../../../../constants/lines';
+
+const ChengduMetroOutsideFareGates = (props: LineStyleComponentProps<ChengduMetroOutsideFareGatesAttributes>) => {
+    const { id, path, newLine, handlePointerDown } = props;
+
+    const onPointerDown = React.useCallback(
+        (e: React.PointerEvent<SVGElement>) => handlePointerDown(id, e),
+        [id, handlePointerDown]
+    );
+
+    return (
+        <path
+            id={id}
+            d={path}
+            fill="none"
+            stroke="#b4b4b5"
+            strokeWidth="5"
+            strokeDasharray="6 5"
+            cursor="pointer"
+            onPointerDown={newLine ? undefined : onPointerDown}
+            pointerEvents={newLine ? 'none' : undefined}
+        />
+    );
+};
+
+/**
+ * ChengduMetroOutsideFareGates specific props.
+ */
+export interface ChengduMetroOutsideFareGatesAttributes extends LinePathAttributes {}
```

Comment:
If you don't have any specific attributes, you may follow this example :)

https://github.com/railmapgen/rmp/blob/878036be0930d6c4cbaf0a03bbd4b110924eb2a7/src/components/svgs/lines/styles/shmetro-virtual-int.tsx#L23